test/DynamoCoreWpfTests/ViewExtensions/GraphNodeManagerViewExtensionTests.cs

```diff
@@ -211,10 +211,12 @@ public void ContainsEmptyListOrNullTest()
             int emptyListNodesCount = hwm.Nodes.Count(ContainsAnyEmptyLists);
 
             var view = viewExt.ManagerView;
-            var images = WpfUtilities.ChildrenOfType<Image>(view.NodesInfoDataGrid);
 
+            IEnumerable<Image> images = [];
             Utility.DispatcherUtil.DoEventsLoop(() =>
             {
+                images = WpfUtilities.ChildrenOfType<Image>(view.NodesInfoDataGrid);
+
```

Comment:
This is an attempt to stabilize the test. I merged it here because seems to be able to reproduce it quite consistently